aspy/refactor_imports/classify.py

```diff
@@ -105,14 +107,20 @@ def classify_import(
     :param text module_name: The dotted notation of a module
     :param tuple application_directories: tuple of paths which are considered
         application roots.
+    :param tuple override_absent_application_modules: tuple of module names
+        that are considered application modules.
     """
+
     # Only really care about the first part of the path
     base, _, _ = module_name.partition('.')
+
```

Comment:
leave the newlines how they were

Reply:
When will I learn to always `git diff master`?!